bridges/FacebookBridge.php

```diff
@@ -659,16 +659,10 @@ private function collectUserData(){
 							$date = 0;
 						}
 
-						// Build title from username and content
-						$title = $author;
-
-						if(strlen($title) > 24)
-							$title = substr($title, 0, strpos(wordwrap($title, 24), "\n")) . '...';
-
-						$title = $title . ' | ' . strip_tags($content);
-
+						// Build title from content
+						$title = strip_tags($post->find('.userContent')[0]->innertext);
```

Comment:
`find` supports indexing results:

```suggestion
						$title = strip_tags($post->find('.userContent', 0)->innertext);
```